### DISCUSSION TITLE
Homekit-Taster

### DISCUSSION THREAD

**Discussion by burningbuell** [thkl/hap-homematic#325] [Q&A] [answered]
Hallo, 

ich würde gerne einen Taster in HomeKit einbinden, mit dem ich dann ein Programm in der Homematic triggern kann. Leider finde ich nichts darüber.
Kann mir jemand helfen, ob das möglich ist?

**Comment by thkl** [comment, marked as answer]
https://github.com/thkl/hap-homematic/wiki/programs
